opba-auth-rest-api/src/main/resources/static/tpp_auth_api.yml

```diff
@@ -126,14 +145,25 @@ components:
           schema:
             type: string
         X-Request-ID:
-          $ref: '#/components/parameters/X-Request-ID'
+          $ref: "#/components/parameters/X-Request-ID"
+      #NO RESPONSE BODY
+
+    202_RenewalAuthorizationSessionKey:
+      description: User logged in
+      headers:
+        X-Request-ID:
+          $ref: "#/components/parameters/X-Request-ID"
+        Set-Cookie:
+          schema:
+            type: string
+            example: Authorization-Session-ID=abcde12345; Path=/; HttpOnly
       #NO RESPONSE BODY
 
     202_UserAccepted:
       description: User logged in
       headers:
         X-Request-ID:
-          $ref: '#/components/parameters/X-Request-ID'
+          $ref: "#/components/parameters/X-Request-ID"
```

Comment:
Double quotes process '\n' characters in string as if it would be new line, this is not what is desired in URL path references, please use `'` for quoting

Reply:
see above